.changeset/itchy-cows-hunt.md

```diff
@@ -0,0 +1,5 @@
+---
+"@localyze-pluto/components": minor
```

Comment:
Why is this a minor? We are not changing any properties to it.

Reply:
because we are adding functionality (a new variant) in a backward compatible way